test/ProblemDetailsResponseFactoryTest.php

```diff
@@ -32,27 +32,27 @@ class ProblemDetailsResponseFactoryTest extends TestCase
 {
     use ProblemDetailsAssertionsTrait;
 
-    /** @var ServerRequestInterface|ObjectProphecy */
+    /** @var ServerRequestInterface|MockObject */
     private $request;
 
-    /** @var ResponseInterface|ObjectProphecy */
+    /** @var ResponseInterface|MockObject */
     private $response;
 
     /** @var ProblemDetailsResponseFactory */
     private $factory;
 
     private const UTF_8_INVALID_2_OCTET_SEQUENCE = "\xc3\x28";
 
-    protected function setUp() : void
+    protected function setUp(): void
```

Comment:
There are about 26 phpunit errors left in this file and I don't know where it is coming from:

> Return value of PHPUnit\Framework\Constraint\Callback::matches() must be of the type bool, string returned

@weierophinney any idea what I miss here?